### PR TITLE
Expand Jenkins variable in taskTemplate clause.

### PR DESCRIPTION
**Pull request by ws171913**
Jenkins variables are provided in form eg. ${env.BUILD_ID}:

You can use Image built in previous steps like this
        taskTemplate(
            name: 'test-my-image',
            image: 'img-repo:5003/my/image:${env.BRANCH_NAME}_${env.BUILD_ID}',
            resourcesMemory: 2048,
            resourcesCPU: 1000
        )

### CODE REVIEW

**Comment by multani** [comment]
@ws171913 Thanks for the PR!

I know the documentation is lacking but you would mind adding some bits about what your change would provide in https://github.com/multani/nomad-pipeline/blob/master/README.md#list-of-settings?
That would be great!

**Comment by multani** [comment]
Can you rebase on top of master to make Travis happy?

**Comment by ws171913** [comment]
No need to rebase as you see :) 
Git managed to apply all changes wisely

File: README.md [hunk 1]
```diff
@@ -153,6 +153,43 @@ TODO: complete me!
 
 ### `taskTemplate`
 
+* `name`: name of Nomad task group to run. It specifies value for
+  `TaskGroups/Name` in Nomad Job definition.
```
Comment (multani) [inline README.md:157]: I would link directly to the relevant Nomad job specification configuration in that case (which would be https://nomadproject.io/docs/job-specification/group/ , although there's no direct `name` attribute for a group.)

File: README.md [hunk 2]
```diff
@@ -153,6 +153,43 @@ TODO: complete me!
...
+* `image`: image identifier to use for the job. Specifies value for
+  `TaskGroups/Tasks/Config/image` in Nomad Job definition.
```
Comment (multani) [inline README.md:159]:
```suggestion
* `image`: Docker image identifier to use for the job. Specifies value for
  [a task's Docker image](https://nomadproject.io/docs/drivers/docker/#inlinecode-image-16) in 
  a Nomad job definition.
```

**Review by multani — APPROVED** [review]
Thanks for the PR @ws171913 !

I'll make a few updates on the README but I think it's cool!

**Comment by multani** [comment]
Thanks a lot @ws171913 for the PR!

**Pull request merged**